dashboard/src/js/components/Header.js

```diff
@@ -1,23 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
Would we eventually be able to separate these components into a CIRG-wide library? just curious...

Reply:
yeah, that will be a great direction.  something like this maybe:  https://bit.dev/frontend-teams